LoopUI/Views/CGMStatusHUDView.swift

```diff
@@ -63,15 +63,15 @@ public final class CGMStatusHUDView: DeviceStatusHUDView, NibLoadable {
         statusStackView.removeArrangedSubview(glucoseValueHUD)
         statusStackView.removeArrangedSubview(glucoseTrendHUD)
         
-        super.presentAlert()
+        super.presentStatusHighlight()
     }
     
-    override public func dismissAlert() {
-        guard statusStackView.arrangedSubviews.contains(alertStatusView) else {
+    override public func dismissStatusHighlight() {
```

Comment:
👍 Nice.  I almost commented in your last PR that using the word "Alert" might be confusing, but I figured it wasn't even worth a [nit]. Thanks for changing this. :)

Reply:
It was @ps2 suggestion.